models/wishlistDao.js

```diff
@@ -0,0 +1,59 @@
+const appDataSource = require('./dataSource');
+
+const getWishlist = async (userId) => {
+  const getWishlist = await appDataSource.query(`
+    SELECT
+      u.id AS userId,
+      p.id AS productId,
+      p.price AS price,
+      p.name AS name,
+      p.thumbnail_image_url AS image,
+      lg.name AS locationName
+    FROM wishlists w
+    INNER JOIN users AS u ON w.user_id = u.id
+    INNER JOIN products AS p ON w.product_id = p.id
+    INNER JOIN location AS l ON p.location_id = l.id
+    INNER JOIN location_groups AS lg ON l.location_group_id = lg.id
+    WHERE w.user_id = ${userId}
```

Comment:
Alias(별칭) 선언 방식을 통일해주세요!
`AS` 키워드를 사용한다면 Alias를 사용하는 부분에 `AS`키워드르 사용해주세요. 
```suggestion
    SELECT
      u.id AS userId,
      p.id AS productId,
      p.price AS price,
      p.name AS name,
      p.thumbnail_image_url AS image,
      lg.name AS locationName
    FROM wishlists AS w
    INNER JOIN users AS u ON w.user_id = u.id
    INNER JOIN products AS p ON w.product_id = p.id
    INNER JOIN location AS l ON p.location_id = l.id
    INNER JOIN location_groups AS lg ON l.location_group_id = lg.id
    WHERE w.user_id = ${userId}
```